etrago/appl.py

```diff
@@ -286,25 +303,40 @@ def run_etrago(args, json_path):
         in sklearn-package (sklearn/cluster/k_means_.py).
         This function doesn't work together with ``'line_grouping = True'``.
 
+    sector_coupled_clustering : nested dict
+        {'active': True, 'carrier_data': {
+         'H2_ind_load': {'base': ['H2_grid'], 'skip': None, },
+         'central_heat': {'base': ['CH4'], 'skip': 'central_heat_store'},
+         'rural_heat': {'base': ['CH4'], 'skip': 'rural_heat_store'}}
+        }
+        State if you want to apply clustering of sector coupled carriers, such
+        as central_heat or rural_heat. The approach builds on already clustered
+        buses (e.g. CH4 and AC) and builds clusters around the topology of the
+        buses with carrier ``'base'`` for all buses of a specific carrier, e.g.
+        ``'H2_ind_load'``. Connections (via links) of the selected buses to
+        buses of the carrier specified in ``'skip'`` are ignored in the
+        clustering and aggregated based on the newly clustered buses in a
+        following step.
+
```

Comment:
Sorry, from this explanation I do not get the 'skip' functionality. Can you explain it to me? And maybe it is worth it to reformulate the given definition - at least if you wanna reach the DAU ;)

Reply:
I'll try to rephrase that, thank you for the comment.

The idea is, that the `bus0` of all links (e.g. heat pump, gas boiler, resistive heater, ...) connected to the sector coupled buses (e.g. the central heat buses) are collected. Then, for each sector coupled bus, I check which other buses have the same combination of `bus0` on the connected links. Buses with the same combination are then aggregated. For this, the storage buses have to be ignored/skipped in the first step, since every heat bus has an own storage bus, which does mean that the combination of `bus0`s including the storage bus is unique for every heat bus.

Therefore, after the aggregation of the heat buses, the heat storages are aggregated according to the newly created busmap.

Reply:
I think I roughly understand it now. It sounds like, that the skipping is really s.th. which should be done always with respect to the strategy (e.g. 'central_heat': {'base': ['CH4']})? Do you really want to give this skip option to the user or is it s.th. rather to adjust in the function itself because it is rather a theoretic thing to do for advanced users? At least I do not get when you would not want to use the skipping function...

Reply:
Good point, I will remove the option and just handle it directly.